src/web100srv.c

```diff
@@ -1403,11 +1403,13 @@ void child_process(int parent_pipe, SSL_CTX *ssl_context, int ctlsockfd) {
   tcp_stat_agent *agent;
   Connection ctl = {-1, NULL};
   ctl.socket = ctlsockfd;
+
   // this is the child process, it handles the connection with the client and
   // runs the actual tests.
   // First, start the watchdog timer
   alarm(300);
```

Comment:
Add a comment explaining where we expect the alarm handler was installed.
Let's discuss offline whether the alarm handler should be installer here.